tests/unit/store/traction/ont/contracts.js

```diff
@@ -60,10 +60,49 @@ const storeData = {
   },
```

Comment:
Saw this idea of contracts. Interesting to know the purpose?

Reply:
The idea is to create organised, consistent test data across files. However I am not a big fan of it, I think the route issue is the amount and quality of our test data, not so much the organisation. So could be changed / removed